src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ReviewOrderContentProvider.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.staff.models.PersonContent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.ReviewOrder;
+
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+@Component
+public class ReviewOrderContentProvider {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd MMMM yyyy");
+    private static final String CLAIM_REFERENCE_NUMBER = "claimReferenceNumber";
+    private static final String PARTY_ADDRESS = "partyAddress";
+    private static final String PARTY_FULL_NAME = "partyFullName";
+    private static final String REVIEW_REASON = "reviewReason";
+    private static final String REVIEW_DATE = "reviewDate";
+
+
+    public ReviewOrderContentProvider() {
+    }
+
+    public Map<String, Object> createContent(Claim claim) {
+        ReviewOrder reviewOrder = claim.getReviewOrder().orElseThrow(IllegalStateException::new);
+        PersonContent party = reviewOrder.getRequestedBy().equals(ReviewOrder.RequestedBy.DEFENDANT) ?
+           PersonContent.builder()
+               .address(claim.getClaimData().getDefendant().getAddress())
+                .fullName(claim.getClaimData().getDefendant().getName()).build() :
+            PersonContent.builder()
+            .address(claim.getClaimData().getClaimant().getAddress())
+            .fullName(claim.getClaimData().getClaimant().getName()).build();
```

Comment:
Looks like the code formatter hasn't kicked in - please run manually

Reply:
👍